hack/release-scripts/get-latest-sidecar-images

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# ---
+# This script generates a copy of image-digests-template.yaml with the latest tags and associated manifest digests
+# for each sidecar image at OUTPUT_FILEPATH.
+
+# --- Environment Variables
+export ROOT_DIRECTORY IMAGE_DIGESTS_TEMPLATE_FILEPATH OUTPUT_FILEPATH
+ROOT_DIRECTORY=${ROOT_DIRECTORY:=$(git rev-parse --show-toplevel)}
```

Comment:
How does this work with "stacked" git directories? Would it be better to base paths off the directory the script is in?

(also applicable in other file)

Reply:
> Would it be better to base paths off the directory the script is in?

Great corner case, will replace with something like `SCRIPT_PATH=$(dirname $(realpath -s $0))`